src/retrace/retrace.py

```diff
@@ -1348,9 +1347,7 @@ def run_crash_cmdline(self, crash_start: List[str], crash_cmdline: str) -> Tuple
         returncode = 0
         crashdir = self.get_crashdir()
         try:
-            t = 3600
-            if CONFIG["ProcessCommunicateTimeout"]:
-                t = CONFIG["ProcessCommunicateTimeout"]
+            t = CONFIG["ProcessCommunicateTimeout"]
```

Comment:
While we're at it, could you please change the variable name to something meaningful, such as `timeout`? One-letter names irk Pylint.

Reply:
done